### ISSUE TITLE
Module in DNN 9.02 not Displaying

### ISSUE THREAD

**Issue by brandondev1** [cantarus/PolyDeploy#55]
I am unable to see any functionality in the module when I install it in DNN.  When I go to the page I have added the module on, it doesn't display properly.  
![PolyDeployScreenShot](https://user-images.githubusercontent.com/18074719/55576086-6c3a4f80-56de-11e9-9039-06dff195fec6.PNG)

**Comment by bdukes** [comment]
@brandondev1 I agree, this initial experience could definitely be better (I think it's adjusted in the upcoming 1.0 release), but the issue here is typically that you've installed the module onto a remote server, and so your browser session isn't on the IP whitelist, thus you don't have access to the module's functionality.

The simplest fix for this is to access the module directly on the server, so that you hit the localhost exception in the IP whitelist.  However, if you don't have ready access to that server, you can insert an entry into the IP whitelist via SQL.  The tables to view are mentioned in https://github.com/cantarus/PolyDeploy/issues/18#issuecomment-374202622

Hope it helps!

**Comment by brandondev1** [comment]
That allows me to see the extension install functionality, but I was expecting the UI for creating users to be there as well.  How do you create users with an API Key and Encryption Key?  Setting up CI builds, so I need to be able to install extensions via the remote install method. 

![PolyDeployScreenShot](https://user-images.githubusercontent.com/18074719/55577003-8c6b0e00-56e0-11e9-92c0-fb92c3bf4312.PNG)

**Comment by bdukes** [comment]
For that functionality, you need to go into edit mode and access the admin screen from the module's action menu.

Hope it helps!
